src/Core/Domain/Product/Combination/Command/UpdateCombinationStockCommand.php

```diff
@@ -0,0 +1,234 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Product\Combination\Command;
+
+use DateTimeInterface;
+use PrestaShop\PrestaShop\Core\Domain\Product\Combination\ValueObject\CombinationId;
+
+/**
+ * Updates combination stock information
+ */
+class UpdateCombinationStockCommand
+{
+    /**
+     * @var CombinationId
+     */
+    private $combinationId;
+
+    /**
+     * @var int|null
+     */
+    private $quantity;
+
+    /**
+     * @var int|null
+     */
+    private $minimalQuantity;
+
+    /**
+     * @var string|null
+     */
+    private $location;
+
+    /**
+     * @var int|null
+     */
+    private $lowStockThreshold;
+
+    /**
+     * @var bool|null
+     */
+    private $lowStockAlertOn;
+
+    /**
+     * @var DateTimeInterface|null
+     */
+    private $availableDate;
+
+    /**
+     * @var bool
+     */
+    private $movementOn = true;
```

Comment:
```suggestion
    private $addMovement = true;
```

To be consistent with `UpdateProductStockInformationCommand` and I feel like it better reflects the action behind it

Reply:
yeah just idk how to call the getter then :sweat:

Reply:
In the product command I just named it `addMovement()`

Reply:
quoting @jolelievre in slack discussion as final decision:
```
so from the search it seems like in legacy AdminProductController and in the migrate ProductController both use StockAvailable::setQuantity which adds a movment by default (without any reason)
so I guess we can just remove the boolean and force the movement all the time
```